modules/ROOT/pages/analyzing-documents-with-einstein.adoc

```diff
@@ -0,0 +1,72 @@
+= Analyzing Documents and Customizing the Output With Einstein
+
+Use the Generic document action with Einstein to fully customize the fields and tables extracted from your documents.  
+
+include::partial$einstein-model.adoc[]
+
+include::partial$document-action.adoc[tag=definition] 
+
+After you create a document action, add reviewers and publish it to Anypoint Exchange. Publishing the document action enables RPA to execute it and also creates an API that you can call from any external system to analyze your documents. 
+
+For instructions about using Einstein in your document actions that use pre-built schemas, see xref:creating-document-actions.adoc[]
+
+== Before You Begin Using Einstein
+
+. Ensure you have any of the following Anypoint permissions:
++
+include::partial$permissions.adoc[tag=permissionManage]
++
+include::partial$permissions.adoc[tag=permissionBuild]
+
+include::partial$einstein.adoc[tags=einsteinRequisites;!shortIntro]
+
+== Create a Generic Document Action and Enable Einstein
+
+To analyze documents with Einstein, create a document action of the Generic type and Enable Einstein:
+
+. In the sidebar, click *Document Actions*.
+. Click *Create New*.
+. Select the *Generic* type. 
+. Enable *Use Einstein*. 
+. Provide a Name and click *Create*.
+
+Next, add the Fields and Tables to extract. 
```

Comment:
Wouldn't it be appropriate to add another point that reads: 
". Add the Fields and Tables to extract."